en-US/ranking/criteria.md

```diff
@@ -2,78 +2,173 @@
+* **Scroll Velocity changes should reflect the music.** Try to avoid changing SV when the must doesn’t call for it. This will generally make your map more difficult to read and harder to enjoy.
 
-The following pertains to the spread of difficulty each mapset must follow.
+## Difficulty Spread
 
-##### Difficulty Names & Description
+| Length        | Minimum Required Maps |
+| ------------- | :-------------------: |
+| **0:45-2:29** |           2           |
+| **2:30+**     |           1           |
 
-The standard names for difficulties in a mapset are Beginner, Easy, Normal, Hard, Insane, and Expert.
-Should you choose not to use the standard names, the difficulty names in your set must show clear sign of progression in difficulty consistent with that of the mapset and should be easy to interpret.
+#### Rules
 
-The following descriptions are what each difficulty should typically look like. Keep in mind that this isn't concrete, and there is room for subjectivity in individual cases. It is only to serve a general outlook on how each map should look and feel.
+* The standard names for difficulties in a mapset are **Beginner, Easy, Normal, Hard, Insane, and Expert**. Should you choose not to use the standard names, the difficulty names in your set must show clear sign of progression in difficulty consistent with that of the mapset and should be easy to interpret. The highest difficulty of each game mode is always free to have a custom difficulty name.
+     * Standard difficulty name may be added to the end of custom difficulty name in brackets to improve the interpretation of difficulty.
+* You are **not** permitted to skip difficulties for **sets with a song length between 0:45-2:29.** For instance, if you are making a 1 minute set with an Insane, you can either create a Hard or an Expert difficulty. Having a Beginner and an Insane is not allowed. **This rule does not apply to sets with a song length of 2:30 or longer.**
+* **Each game mode is treated separately.** If you are creating a mapset for both 4 Keys and 7 Keys, you must have a difficulty spread for both game modes.
+* **If a mapset has two game modes, each difficulty must be prefaced with either `4K` or `7K`** (Example: `7K Insane`).
+* **If a mapset contains guest difficulties, the guest difficulties creators must be credited in their respective difficulty names** (Example: `Flamingo's Hard`). Mapset's host should not be credited in this fashion.
+* **The number of difficulties created by the host of a mapset must be equal or higher than the largest number of guest difficulties created by a single user.** If a mapper with guest difficulties has contributed to the set with more difficulties than the host, the host should be changed. The creators of guest difficulties should also be credited in the tags.
```

Comment:
This suggested change would cause tags to become inconsistent across difficulties, conflicting with the rule: `**Metadata must be consistent across difficulties.** The only exception being specific map-descriptive tags, such as pattern names, that improve the searchability of singular difficulties.`

Either another exception needs to be made in the quoted rule, or guest creators need to be tagged in every map of the mapset I guess.